src/mocks/handlers.ts

```diff
@@ -64,13 +64,6 @@ const handlers = [
   rest.get(ipLookupApiUrl, (req, res, ctx) => {
     return res(ctx.status(200), ctx.json(ipLookupData), ctx.delay(500));
   }),
-
```

Comment:
**Nitpick** Maybe try to add new things at the end of file, because just by looking at PR I thought you deleted 2 handlers

Reply:
These handlers were identical to the other two above.

Reply:
Okay. Then maybe I should pay more attention, because at first it looked like you have deleted them